lib/src/main/java/formflow/library/utils/HashMapConverter.java

```diff
@@ -0,0 +1,41 @@
+package formflow.library.utils;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
```

Comment:
Is Jackson only being used for the exception in this case?

Reply:
The ObjectMapper is from Jackson as well.